userspace/libscap/scap.c

```diff
@@ -1581,6 +1581,21 @@ static int32_t scap_next_nodriver(scap_t* handle, OUT scap_evt** pevent, OUT uin
 }
 #endif // _WIN32
 
+#ifdef _WIN32
+inline uint64_t get_windows_timestamp()
+{
+	FILETIME ft;
+	static const uint64_t EPOCH = ((uint64_t) 116444736000000000ULL);
+
+	GetSystemTimePreciseAsFileTime(&ft);
+
+	uint64_t ftl = (((uint64_t)ft.dwHighDateTime) << 32) + ft.dwLowDateTime;
+	ftl -= EPOCH;
+
+	return ftl * 100;
+}
+#endif // _WIN32
+
```

Comment:
I'd rather:
1. have OS-specific files instead of OS-specific code interleaved with platform-agnostic
2. make the OS-specific functions compatible (i.e. use the same API across platforms) so that we don't need more `#ifdef`s around call sites (i.e. how is this different from sinsp_utils::get_current_time_ns()? ...except for being in a different library, sigh)
3. add this function together with its first user

Reply:
Actually, I don't see this being used anywhere in the projects using libs, so let me see if I can remove it.